load_balancing.go

```diff
@@ -26,6 +26,7 @@ type LoadBalancerPool struct {
 	Longitude         *float32                    `json:"longitude,omitempty"`
 	LoadShedding      *LoadBalancerLoadShedding   `json:"load_shedding,omitempty"`
 	OriginSteering    *LoadBalancerOriginSteering `json:"origin_steering,omitempty"`
+	Healthy           bool                        `json:"healthy,omitempty"`
```

Comment:
this is what semgrep failure is talking about - https://github.com/cloudflare/cloudflare-go/blob/master/docs/conventions.md#booleans

```suggestion
	Healthy           *bool                       `json:"healthy,omitempty"`
```

you'll also need to update your struct usages to be `BoolPtr(true)` or `BoolPtr(false)`